File: src/main/java/TTSService.java
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.Headers;
import retrofit2.http.POST;

public interface TTSService {
	
	
    //@GET("/tasks")
    //Call<List<Task>> getTasks();
	
	@Headers("Access-Token: fdf1d9f7a7fcdc2168eb1ad39fad88ff")
	@POST("/tts")
	Call<Object> send(@Body TTS tts);
	
}

File: src/main/kotlin/com/github/dsaouda/totalvoice/service/TTSService.kt
package com.github.dsaouda.totalvoice.service

import com.github.dsaouda.totalvoice.data.TTS
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface TTSService {

    @POST("/tts")
    fun send(@Body tts: TTS): Call<Any>
}